2-mandatory/4-stocks.js

```diff
@@ -65,6 +86,17 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+    let highestPriceForEachStock = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+    highestPriceForEachStock.push(
+        `The highest price of ${stocks[
+        i
+        ].toUpperCase()} in the last 5 days was ${Math.max(
```

Comment:
Hi Marvin: There is an "Uncaught Reference Error: closingPricesForAllStocks is not defined"